### DISCUSSION TITLE
r/city-china-jiangsu-nantong

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#34] [Announcements]
# r/city-china-jiangsu-nantong

游记 | 南通，值得被认真探索

https://blog.ops-coffee.cn/r/city-china-jiangsu-nantong

**Comment by xyzbz** [comment]
那个气球上面的字好像是棱镜乐队的一首歌

**Reply by ops-coffee** [reply]
我不知道哎，但是很好看

**Comment by DayuGuo** [comment]
这两年因女友工作需求，跟着一起去了好几次南通。读着这篇文章，唤起了好多回忆。
下次再有机会去，会去尝尝茉不作茶！

**Reply by ops-coffee** [reply]
这是个很好的故事

**Comment by beyond1314** [comment]
写的很详细，顶一下
